### PR TITLE
Update local-cluster example to start a 3+1 node cluster

### PR DESCRIPTION
**Pull request by pcholakov**
Also sets default logging to debug for Restate, and bootstrap partitions to 2.

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -119,7 +119,10 @@ run *flags: (_target-installed target)
     cargo run {{ _target-option }} {{ flags }}
 
 test: (_target-installed target)
-    cargo nextest run {{ _target-option }} --all-features
+    cargo nextest run {{ _target-option }} --all-features --target-dir target/tests
+
+test-package package *flags:
+    cargo nextest run --all-features --no-capture --package {{ package }} --target-dir target/tests {{ flags }}
```
Comment (pcholakov) [inline justfile:125]: Unrelated sneaky change - I still want to update the dev docs but this is very useful in the mean time.

**Review by muhamadazmy — APPROVED** [review]
LGTM! ❤️

File: justfile [hunk 2]
```diff
@@ -119,7 +119,10 @@ run *flags: (_target-installed target)
     cargo run {{ _target-option }} {{ flags }}
 
 test: (_target-installed target)
-    cargo nextest run {{ _target-option }} --all-features
+    cargo nextest run {{ _target-option }} --all-features --target-dir target/tests
```
Comment (tillrohrmann) [inline justfile:122]: Why did you change the target-dir here @pcholakov?
